Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_SIEM_search_for_failed_logon_-_Generic.yml

```diff
@@ -0,0 +1,706 @@
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: f2ec99bf-b674-4519-8a60-e9440a096f1f
+    type: condition
+    task:
+      id: f2ec99bf-b674-4519-8a60-e9440a096f1f
+      version: -1
+      name: Found results from Azure Log Analytics?
+      description: Check if there are results from Azure Log Analytics
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "22"
+      "yes":
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: AzureFailedLogonLogs.tables.rows.[0]
+                accessor: '[0]'
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 820,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: 3f65bbaf-fa67-4cc7-84a7-49d625b95bef
+    type: title
+    task:
+      id: 3f65bbaf-fa67-4cc7-84a7-49d625b95bef
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -440,
+          "y": 1610
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "16_22_#default#": 0.11,
+      "17_22_#default#": 0.26,
+      "18_22_#default#": 0.1,
+      "18_9_yes": 0.71
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1525,
+        "width": 1880,
+        "x": -440,
+        "y": 150
+      }
+    }
+  }
+inputs:
+- key: SplunkIndex
+  value: {}
+  required: false
+  description: Splunk's index name in which to search. Default is "*" - All.
+  playbookInputQuery:
+- key: SplunkEarliestTime
+  value: {}
+  required: false
+  description: The earliest time for the Splunk search query.
+  playbookInputQuery:
+- key: SplunkLatestTime
+  value: {}
+  required: false
+  description: The latest time for the Splunk search query.
+  playbookInputQuery:
+- key: Username
+  value: {}
+  required: false
+  description: User name.
+  playbookInputQuery:
+outputs:
+- contextPath: NumOfSiemFailedLogon
```

Comment:
Let's add the SIEM results as an output as well

Reply:
Done.